plugins/Jobs.cpp

```diff
@@ -1000,6 +985,29 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
             STHROW("502 Failed to update job data");
         }
 
+        // If this was the last queued child, resume the parent
+        SQResult result;
+        if (!db.read("SELECT parentJobID "
+                     "FROM jobs "
+                     "WHERE jobID=" + SQ(jobID) + ";",
+                     result)) {
+            STHROW("502 Select failed");
+        }
+        const string& safeParentJobID = SQ(result[0][0]);
```

Comment:
Shouldn't you check that safeParentJobID isn't 0?

Reply:
If the parent is 0, we will never reach this point, we cannot cancel jobs without a parent, in the peek part of the command